pkg/utils/apiresources.go

```diff
@@ -81,7 +93,10 @@ func NewResourceConfig(isAllowList bool) *ResourceConfig {
 	r.AddGroup(metricsV1beta1.GroupName)
 	r.AddGroupVersionKind(corev1PodGVK)
 	r.AddGroupVersionKind(corev1NodeGVK)
+	// disable networking resources
```

Comment:
should add the entire fleet-network group instead of one by one

Reply:
it depends. for example, the serviceExport and mcs should be placed to the members.

Need to check if we support disabling all networking gvk but excluding the  serviceExport and mcs today.